### ISSUE TITLE
iOS Code Signing Error

### ISSUE THREAD

**Issue by h0shy** [comScore/Comscore-Swift-Package-Manager#11]
Since version 6.10.1, we cannot submit apps to the iOS Appstore anymore (see screenshot). It seems that `strip-framework.sh` is included in the final app bundle. Since shell scripts cannot be signed, the App Store distribution checks fail.

<img width="562" alt="Screenshot 2023-06-14 at 14 26 30" src="https://github.com/comScore/Comscore-Swift-Package-Manager/assets/30653817/95af6959-fbc5-455a-bd3b-1e9200da11e0">

**Comment by pyby** [comment]
Same issue on SRGSSR products, when updating SPM dependencies and updating ComScore from 6.10.0 to 6.10.1.
Submitting a build to AppStore Connect:

- `tvOS` platform: ✅
- `iOS` platform: ❌

```
[Application Loader Error Output]: ERROR: [ContentDelivery.Uploader] Asset validation failed (90035) Invalid Signature. Code object is not signed at all. The file at path “[$APP].app/Frameworks/ComScore.framework/strip-framework.sh” is not properly signed. Make sure you have signed your application with a distribution certificate, not an ad hoc certificate or a development certificate. Verify that the code signing settings in Xcode are correct at the target level (which override any values at the project level). Additionally, make sure the bundle you are uploading was built using a Release target in Xcode, not a Simulator target. If you are certain your code signing settings are correct, choose “Clean All” in Xcode, delete the “build” directory in the Finder, and rebuild your release target. For more information, please consult https://developer.apple.com/support/code-signing. (ID: a57fc692-a10f-49e1-a3f2-858e2e710556)
[Application Loader Error Output]: Error uploading '/Users/[$USER]/buildAgent/temp/buildTmp/[$API_UUID].ipa'.
[Application Loader Error Output]: The call to the altool completed with a non-zero exit status: 1. This indicates a failure.
```

In [Github release 6.10.1](https://github.com/comScore/Comscore-Swift-Package-Manager/releases/tag/v6.10.1) `http://ComScore.xcframework.zip` file:
- `strip-framework.sh` is introduced on `iOS` platform, like `ios-arm64/ComScore.framework`.
- Nothing change for `tvOS` platform, like in `tvos-arm64/ComScore.framework`.

Does the script should be removed from iOS platform and its variants?

**Comment by JoseCabo-comscore** [comment]
Thank you very much for reporting this issue. Something went wrong in our side due to an unexpected folder structure change that caused the `strip-framework.sh` to remain instead of being deleted as it should have. The issue have been resolved if you update to 6.10.2.

However, note that the binaries of 6.10.1 have been re-used and Comscore SDK will transmit as such.

**Resolution**:
Please, update to the new release 6.10.2.

**Comment by pyby** [comment]
@JoseCabo-comscore Thank you. Fixed.
